pkg/dao/repository_config_test.go

```diff
@@ -967,13 +967,13 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 			URL:  pointy.String(""),
 		}, { // Providing a valid url & name
 			Name: pointy.String("Some Other Name"),
-			URL:  pointy.String("http://foobar.com"),
+			URL:  pointy.String("http://foobar.com/"),
 		}, { // Providing a bad url that doesn't have a repo
 			Name: pointy.String("Some bad repo!"),
-			URL:  pointy.String("http://badrepo.com"),
+			URL:  pointy.String("http://badrepo.com/"),
```

Comment:
"badrepo.com
is parked free, courtesy of GoDaddy.com."
Can you use 
```suggestion
			URL:  pointy.String("http://badrepo.test/"),
```